core/src/main/kotlin/fr/sncf/osrd/stdcm/graph/STDCMPostProcessing.kt

```diff
@@ -131,73 +128,56 @@ private fun makeBlockWaypoints(
 }
 
 /**
- * Builds the actual list of EdgeRange given the raw result of the pathfinding. We can't use the
+ * Builds the actual list of STDCMEdge given the raw result of the pathfinding. We can't use the
  * pathfinding result directly because we use our own method to keep track of previous nodes/edges.
  */
-private fun makeEdgeRange(
-    raw: Pathfinding.Result<STDCMEdge, STDCMEdge>
-): List<EdgeRange<STDCMEdge, STDCMEdge>> {
+private fun invertEdges(raw: Result): List<STDCMEdge> {
```

Comment:
This doesn't *just* invert the edges, otherwise we would just call `.reversed`.

It used to build the edge list by iterating over the previous nodes backwards, but I don't think this is required anymore. We can just use `raw.edges`.

Reply:
done